src/rust/engine/src/nodes.rs

```diff
@@ -340,27 +343,32 @@ impl ExecuteProcess {
       .transpose()?;
 
     let output_files = externs::getattr::<Vec<String>>(value, "output_files")
-      .unwrap()
+      .map_err(|err| format!("Failed to get field `output_files`: {err}"))?
       .into_iter()
       .map(RelativePath::new)
       .collect::<Result<_, _>>()?;
 
     let output_directories = externs::getattr::<Vec<String>>(value, "output_directories")
-      .unwrap()
+      .map_err(|err| format!("Failed to get field `output_directories`: {err}"))?
       .into_iter()
       .map(RelativePath::new)
       .collect::<Result<_, _>>()?;
 
-    let timeout_in_seconds: f64 = externs::getattr(value, "timeout_seconds").unwrap();
+    let timeout_in_seconds: f64 = externs::getattr(value, "timeout_seconds")
+      .map_err(|err| format!("Failed to get field `timeout_seconds`: {err}"))?;
 
     let timeout = if timeout_in_seconds < 0.0 {
       None
     } else {
       Some(Duration::from_millis((timeout_in_seconds * 1000.0) as u64))
     };
 
-    let description: String = externs::getattr(value, "description").unwrap();
-    let py_level = externs::getattr(value, "level").unwrap();
+    let description: String = externs::getattr(value, "description")
+      .map_err(|err| format!("Failed to get field `description`: {err}"))?;
```

Comment:
It's probably worth checking what the default `getattr` error message looks like, to see whether it actually needs to be enriched at all? If it includes the field that was being looked up, then maybe these could all be:
```suggestion
    let description: String = externs::getattr(value, "description")
      .map_err(|err| err.to_string())?;
```
... instead?